forms-flow-bpm/src/main/java/org/camunda/bpm/extension/commons/connector/HTTPServiceInvoker.java

```diff
@@ -59,15 +61,18 @@ public ResponseEntity<String> executeWithParamsAndPayload(String url, HttpMethod
 
     private String getServiceId(String url) {
 
+        Boolean enableCustomSubmission = Boolean.valueOf(integrationCredentialProperties.getProperty("forms.enableCustomSubmission"));
  		if (isUrlValid(url, fetchUrlFromProperty(API_URL))) {
 			return APPLICATION_ACCESS_HANDLER;
 		} else if (isUrlValid(url, fetchUrlFromProperty(BPM_URL))) {
 			return BPM_ACCESS_HANDLER;
 		} else if (isUrlValid(url, fetchUrlFromProperty(ANALYSIS_URL))) {
 			return TEXT_ANALYZER_ACCESS_HANDLER;
+		} else if (enableCustomSubmission) {
```

Comment:
can you move this under a single else if condition and have an extra check for /submission url matching